libs/StatusGoQt/src/StatusGo/Wallet/WalletApi.h

```diff
@@ -19,7 +20,20 @@ namespace Status::StatusGo::Wallet
 /// \brief Retrieve a list of derived account addresses
 /// \see \c generateAccountWithDerivedPath
 /// \throws \c CallPrivateRpcError
-DerivedAddresses getDerivedAddressesForPath(const HashedPassword &password, const Accounts::EOAddress &derivedFrom, const Accounts::DerivationPath &path, int pageSize, int pageNumber);
+DerivedAddresses getDerivedAddressesForPath(const HashedPassword &password,
```

Comment:
It would be nice to be consistent about & placement. Here it goes together with a variable name, below it goes with a type.

Reply:
I strongly believe we should setup clang-format to avoid code-style wars :) 
For now I'll change  all my changes to `type &name' (despite i prefer `Type& name`)